SmartTutor/Controllers/ContentRepositoryService.cs

```diff
@@ -0,0 +1,129 @@
+            educationContent.EducationDifficulty = 5;
+            educationContent.EducationSnippets = new List<EducationSnippet>();
+
+            EducationSnippet snippetOne = new EducationSnippet();
+            EducationSnippet snippetTwo = new EducationSnippet();
+            EducationSnippet snippetThree = new EducationSnippet();
+            EducationSnippet snippetFour = new EducationSnippet();
+            EducationSnippet snippetFive = new EducationSnippet();
+
+            snippetOne.SnippetQuality = 4;
+            snippetOne.Tags = new List<Tag>(); snippetOne.Tags.Add(Tag.MustKnow);
+            snippetOne.EducationType = EducationType.ShortText;
+            snippetOne.Content = "This method have problem with bad smell: Long method";
+            snippetOne.SnippetDifficulty = 2;
+
+
+            snippetTwo.SnippetQuality = 5;
+            snippetTwo.Tags = new List<Tag>(); snippetTwo.Tags.Add(Tag.Interesting);
+            snippetTwo.EducationType = EducationType.LongText;
+            snippetTwo.Content = "Lines should not be 150 characters long. Functions should not be 100 lines long. Functions should hardly ever be 20 lines long. Every function in this program was just two, or three, or four lines long.Each was transparently obvious.Each told a story. And each led you to the next in a compelling order.That’s how short your functions should be!";
+            snippetTwo.SnippetDifficulty = 3;
+
+            snippetThree.SnippetQuality = 3;
+            snippetThree.Tags = new List<Tag>(); snippetThree.Tags.Add(Tag.Interesting);
+            snippetThree.EducationType = EducationType.ShortText;
+            snippetThree.Content = "Blocks and Indenting. This implies that the blocks within if statements, else statements, while statements, and so on should be one line long.Probably that line should be a function call.";
+            snippetThree.SnippetDifficulty = 2;
+
+
+            snippetFour.SnippetQuality = 4;
+            snippetFour.Tags = new List<Tag>(); snippetFour.Tags.Add(Tag.MustKnow);
+            snippetFour.EducationType = EducationType.Image;
+            snippetFour.Content = "https://refactoring.guru/images/refactoring/content/smells/long-method-01-3x.png";
+            snippetFour.SnippetDifficulty = 4;
+
+
+            snippetFive.SnippetQuality = 4;
+            snippetFive.Tags = new List<Tag>(); snippetFive.Tags.Add(Tag.MustKnow);
+            snippetFive.EducationType = EducationType.Video;
+            snippetFive.Content = "https://www.youtube.com/watch?v=w_SHQFzOosg";
+            snippetFive.SnippetDifficulty = 4;
+
+
+
+            educationContent.EducationSnippets.Add(snippetOne);
+            educationContent.EducationSnippets.Add(snippetTwo);
+            educationContent.EducationSnippets.Add(snippetThree);
+            educationContent.EducationSnippets.Add(snippetFour);
+            educationContent.EducationSnippets.Add(snippetFive);
+
+
+            return educationContent;
```

Comment:
@vlaksi In general, you should think more about your code. While its ok to have an initial solution look like this, you should not rush for a review without first reviewing the work yourself and looking for refactoring opportunities and the broader design.

Regarding refactoring opportunities, a specific example is the length of these functions and the use of blank lines to clearly separate the multiple things the function is doing.

Regarding the broader design, a specific example is the place of this class in the broader Smart Tutor module. While the class is "by design" meant for temporary use, it still represents a legitimate building block and should be appropriately declared as an InMemoryRepository.

Don't take this the wrong way, it is meant to serve as input for improving your development process.

Reply:
First thank you for so much learning opportunities in real examples.

For this class (ContentRepositoryService). i know i rush, my bad. 
I just think how to make it quickly for Milica, and that is bad, i know.